src/reports/components/report-sections/no-failed-instances-congrats.tsx

```diff
@@ -5,16 +5,30 @@ import * as React from 'react';
 import { InlineImage, InlineImageType } from 'reports/components/inline-image';
 import * as styles from './no-failed-instances-congrats.scss';
 
-export const NoFailedInstancesCongrats = NamedFC('NoFailedInstancesCongrats', () => {
-    return (
-        <div className={styles.reportCongratsMessage}>
-            <div className={styles.reportCongratsHead}>Congratulations!</div>
-            <div className={styles.reportCongratsInfo}>No failed automated checks were found.</div>
-            <InlineImage
-                className={styles.sleepingAda}
-                imageType={InlineImageType.SleepingAda}
-                alt=""
-            />
-        </div>
-    );
-});
+export type NoFailedInstancesCongratsDeps = {
+    customCongratsMessage: String;
+};
+
+export type NoFailedInstancesCongratsProps = {
+    deps: NoFailedInstancesCongratsDeps;
+};
+
+export const NoFailedInstancesCongrats = NamedFC<NoFailedInstancesCongratsProps>(
+    'NoFailedInstancesCongrats',
+    props => {
+        const message = props.deps.customCongratsMessage
+            ? props.deps.customCongratsMessage
+            : "No failed automated checks were found. Continue investigating your website's accessibility compliance through manual testing using Tab stops and Assessment in Accessibility Insights for Web";
```

Comment:
would it be better to pass this text (the "coda" of the congrats message for AI-Web) as a deps the same as we do for AI-Android?
Doing so makes all the cases follow the same pattern, which is easier to understand.

Reply:
The reports always pass in a null value, so it's not really feasible to use a code there. Given the messages, I'm not sure how much would be the "base" and how much would be the "coda". My inclination is to stick with this for now, then (after we update the reports to pass in a value), follow a pattern where we have no default and the string is completely specified by the caller. Happy to chat more about this if you want.